src/chaos-experiments/models/chaosExperimentsManager.js

```diff
@@ -102,6 +107,20 @@ module.exports.getChaosJobExperimentsByJobId = async function (jobId, contextId)
     return databaseConnector.getChaosJobExperimentsByJobId(jobId, contextId);
 };
 
+const buildExperimentResource = (kubernetesChaosConfig, jobId) => {
+    const labels = {
+        [CHAOS_EXPERIMENT_LABELS.APP]: PREDATOR_RUNNER_PREFIX,
+        [CHAOS_EXPERIMENT_LABELS.JOB_ID]: jobId
+    };
+    return {
+        ...kubernetesChaosConfig,
+        metadata: {
+            ...kubernetesChaosConfig.metadata,
+            labels: labels
```

Comment:
this means that if the k8s config labels give you labels, you will delete them. Need to add to existing labels (if exist)